### ISSUE TITLE
Plugin fails on Gradle 3.0

### ISSUE THREAD

**Issue by RaySinnema** [jk1/TeamCity-dependencies-gradle-plugin#16]
```
* What went wrong:
A problem occurred configuring project '<redacted>'.
> Failed to notify project evaluation listener.
> org/gradle/api/internal/artifacts/ivyservice/ivyresolve/strategy/ResolverStrategy
* Exception is:
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project '<redacted>'.
[...]
Caused by: java.lang.NoClassDefFoundError: org/gradle/api/internal/artifacts/ivyservice/ivyresolve/strategy/ResolverStrategy
  at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.getParser(DependenciesRegexProcessor.groovy:93)
  at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.readArtifactsSet(DependenciesRegexProcessor.groovy:86)
```

**Comment by jonnyzzz** [comment]
+1 from me

```
Caused by: java.lang.NoClassDefFoundError: org/gradle/api/internal/artifacts/ivyservice/ivyresolve/strategy/ResolverStrategy
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.getParser(DependenciesRegexProcessor.groovy:93)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.readArtifactsSet(DependenciesRegexProcessor.groovy:86)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor$_processDependencies_closure2.doCall(DependenciesRegexProcessor.groovy:41)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.processDependencies(DependenciesRegexProcessor.groovy:33)
        at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:382)
        at org.gradle.internal.metaobject.BeanDynamicObject.invokeMethod(BeanDynamicObject.java:170)
        at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:80)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor$_process_closure7.doCall(DependenciesRegexProcessor.groovy:141)
        at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:70)
        at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:160)
        at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:106)
        at org.gradle.util.ConfigureUtil$1.execute(ConfigureUtil.java:123)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:117)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:122)
        at org.gradle.api.DomainObjectCollection$all$0.call(Unknown Source)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor.process(DependenciesRegexProcessor.groovy:139)
        at com.github.jk1.tcdeps.processing.DependenciesRegexProcessor$process.call(Unknown Source)
        at com.github.jk1.tcdeps.TeamCityDependenciesPlugin$_apply_closure2.doCall(TeamCityDependenciesPlugin.groovy:44)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:44)
```

**Comment by jk1** [comment]
Fixed in 0.10

**Comment by RaySinnema** [comment]
I just verified that this indeed works. Thanks for the quick turnaround!
